lib/Constants.js

```diff
@@ -180,7 +180,8 @@ module.exports.MessageTypes = {
     GUILD_DISCOVERY_DISQUALIFIED: 14,
     GUILD_DISCOVERY_REQUALIFIED: 15,
     GUILD_DISCOVERY_GRACE_PERIOD_INITIAL_WARNING: 16,
-    GUILD_DISCOVERY_GRACE_PERIOD_FINAL_WARNING: 17
+    GUILD_DISCOVERY_GRACE_PERIOD_FINAL_WARNING: 17,
+    INLINE_REPLY: 19
```

Comment:
This is named `REPLY` in discord side, should probably follow the same convention I guess